addons/product/tests/test_seller.py

```diff
@@ -164,3 +164,28 @@ def test_50_seller_ids(self):
         vendors.write({'product_id': False})
         self.assertEqual(vendors, self.product_consu.seller_ids,
             "Setting the product_id to False shouldn't affect seller_ids.")
+
+    def test_51_seller_ids(self):
+        """Test vendor selection when using select_seller with Partner_id False"""
+        self.ipod_nano = self.env['product.product'].create([{
+            'name': 'The one with all the cool games',
+            'standard_price': 200.0,
+        }])
+
```

Comment:
Use `self.product_consu` instead, the purpose of data created in the setup is to not have to create data in each test individually
```suggestion
```